src/webots/nodes/WbGeometry.hpp

```diff
@@ -129,7 +129,8 @@ class WbGeometry : public WbBaseNode {
 
   // visibility
   void setTransparent(bool isTransparent);
-
+  bool isTransparent() const;
```

Comment:
Such a simple method could be inline:
```suggestion
  bool isTransparent() const { return mIsTransparent; }
```

Reply:
Ok I wasn't sure what was the rules in that case. For performance reasons I would tend to do the same but these days the compilers are quite clever if you use the right parameters.
I'll follow your guidance here, it is your code base ;)